wdio/specs/homepage.js

```diff
@@ -46,25 +66,14 @@ describe('Municipality homepage', () => {
 
   it('Should show "english" language and "residents" chosen in the language and user type menus', () => {
     browser.url('/municipality-1/node/1?language=en&user_type=residents');
-    browser.waitForVisible('.btn-group.user-types');
-    browser.waitForVisible('.btn-group.languages');
-    const userType = browser.getText('.btn-group.user-types .btn.btn-default.active');
-    assert.equal('Residents', userType);
-
-    const languague = browser.getText('.btn-group.languages .btn.btn-default.active');
-    assert.equal('English', languague);
+    checkSelectedLanguage('English');
+    checkSelectedUserType('Residents');
   });
 
   it('should show "hebrew" language and "residents" chosen in the language and user type menus', () => {
     browser.url('/municipality-1/node/1?language=he&user_type=residents');
-    browser.waitForVisible('.btn-group.user-types');
-    browser.waitForVisible('.btn-group.languages');
-
-    const userType = browser.getText('.btn-group.user-types .btn.btn-default.active');
-    assert.equal('תושבים', userType);
-
-    const languague = browser.getText('.btn-group.languages .btn.btn-default.active');
-    assert.equal('עברית', languague);
+    checkSelectedLanguage('עברית');
```

Comment:
I think it would be nicer for these functions to get an english argument. So it would be `checkSelectedLanguage('heberw');` and then in `checkSelectedLanguage` you will have a `switch` to get the values you need.

Like this once `checkSelectedUserType('Businesses AR');` is translated to Arabic, you will have to change it only in a single place.